src/coreComponents/constitutive/contact/ContactBase.hpp

```diff
@@ -71,9 +71,10 @@ class ContactBaseUpdates
    * @return The hydraulic aperture that is always > 0
    */
   GEOSX_HOST_DEVICE
-  inline
-  virtual real64 computeHydraulicAperture( real64 const aperture,
-                                           real64 & dHydraulicAperture_dAperture ) const;
+  real64 computeHydraulicAperture( real64 const aperture, real64 & dHydraulicAperture_dAperture ) const
```

Comment:
This had to be made non-virtual to avoid an issue in the cce device-pass where it would inappropriately inline a virtual function in a context it isn't able to by the C++ standard, which caused a segfault when using this function in device kernels since the "output param" wasn't appropriately bound to a valid memory location.